activiti-cloud-audit-service/activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/AuditVariableEventsIT.java

```diff
@@ -60,8 +56,13 @@ class AuditVariableEventsIT {
     @Autowired
     private MyProducer producer;
 
-    @BeforeEach
-    public void setUp() {
+    //    @BeforeEach
+    //    public void setUp() {
+    //        repository.deleteAll();
+    //    }
```

Comment:
Please, delete commented block. 
Are we sure that the repository is empty at the beginning or in addition to `@AfterEach` as also need to clean in `@BeforeAll` to make sure that no other test suite left data behind?

Reply:
Thank you for pointing it out :)
In this case the repository should be empty because the context used by `AuditVariableEventsIT` has a unique configuration. But I am going to change also `AuditServiceIT` too for using `deleteAll` in `@AfterEach` phase to be absolutely sure.